mllib/src/main/scala/org/apache/spark/ml/feature/MinMaxScaler.scala

```diff
@@ -111,7 +111,7 @@ class MinMaxScaler @Since("1.5.0") (@Since("1.5.0") override val uid: String)
 
   @Since("2.0.0")
   override def fit(dataset: Dataset[_]): MinMaxScalerModel = {
-    transformSchema(dataset.schema, logging = true)
+    transformSchema(dataset.schema)
```

Comment:
It's a good question. Since ```MinMaxScaler``` override ```transformSchema``` with no argument ```logging```, we should use that one rather than the function in the base class.